ssr/src/page/wallet/tokens.rs

```diff
@@ -232,16 +233,28 @@ pub fn TokenTile(user_principal: Principal, token_meta_data: TokenMetadata) -> i
                 class="flex flex-1  p-y-4"
             >
                 <div class="flex flex-2 items-center space-x-2 px-2">
-                    <img
-                        class=move || {
-                            let mut classes = "w-12 h-12 rounded-full".to_string();
-                            if info.is_nsfw {
-                                classes.push_str(" blur-md");
+                    <div class="relative">
+                        <img
+                            class=move || {
+                                let mut classes = "w-12 h-12 rounded-full".to_string();
+                                if info.is_nsfw {
+                                    classes.push_str(" blur-md");
+                                }
+                                classes
                             }
-                            classes
-                        }
-                        src=info.logo_b64.clone()
-                    />
+                            src=info.logo_b64.clone()
+                        />
+                        {move || if info.is_nsfw {
```

Comment:
If the extra `<div></div>` is not intentional or doesnt have any significiant eg css as dependency etc, Use `<Show when=condition/>` instead of this it will minimize the need of <div></div> in the else block

example:
```jsx
            <Show when=move || authorized().map(|(a, _)| a).unwrap_or_default()>
                <YoutubeUploadInner canisters=cans_s.get_value() url=url_s.get_value() />
            </Show>
```